### PR TITLE
fix: don't crash on tray.setContextMenu(null)

### PR DESCRIPTION
**Pull request by miniak**
##### Description of Change
Fixes https://github.com/electron/electron/issues/14188

##### Checklist
- [x] PR description included and stakeholders cc'd
- [x] `npm test` passes
- [x] tests are [changed or added](https://github.com/electron/electron/blob/master/docs/development/testing.md)
- [x] relevant documentation is changed or added
- [x] PR title follows semantic [commit guidelines](https://github.com/electron/electron/blob/master/docs/development/pull-requests.md#commit-message-guidelines)

##### Release Notes
Notes: Fixed crash on `tray.setContextMenu(null)`.

### CODE REVIEW

**Comment by miniak** [comment]
@ckerr, @codebytere can you please review?

File: atom/browser/api/atom_api_tray.cc [hunk 1]
```diff
@@ -210,7 +210,7 @@ void Tray::PopUpContextMenu(mate::Arguments* args) {
 
 void Tray::SetContextMenu(v8::Isolate* isolate, mate::Handle<Menu> menu) {
   menu_.Reset(isolate, menu.ToV8());
-  tray_icon_->SetContextMenu(menu->model());
+  tray_icon_->SetContextMenu(menu.IsEmpty() ? nullptr : menu->model());
```
Comment (alexeykuzmin) [inline atom/browser/api/atom_api_tray.cc]: Can `menu->model()` just return a `nullptr` when it's empty?

Comment (miniak) [reply atom/browser/api/atom_api_tray.cc]: menu itself is empty reference

Comment (alexeykuzmin) [reply atom/browser/api/atom_api_tray.cc]: Oh, right.

**Comment by ckerr** [comment]
Merging this PR -- these CI failures are known glitches & are not related to this PR.

**Pull request merged**